src/core_plugins/vega/public/data_model/vega_parser.js

```diff
@@ -320,7 +361,10 @@ export class VegaParser {
     if (val === undefined) {
       dstObj[paramName] = dflt;
     } else if (typeof val !== 'boolean') {
-      this._onWarning(`config.kibana.${paramName} must be a boolean value`);
+      this._onWarning(i18n.translate('vega.vegaParser.someKibanaParamValueTypeWarningMessage', {
+        defaultMessage: 'config.kibana.{paramName} must be a boolean value',
```

Comment:
Let's separate config name `config.kibana.${paramName}` into `values` instead of just `paramName`.